### DISCUSSION TITLE
Issue Deleted?

### DISCUSSION THREAD

**Discussion by kmcclive** [aws/aws-cdk#25317] [General]
I opened an issue (25301) yesterday that has since been deleted.  I am guessing it was considered a vulnerability.  @peterwoodworth I saw that you commented on it before deleting it.  Is there another way that I can track it?  Thanks in advance.

**Comment by peterwoodworth** [comment]
we fixed the bug https://github.com/aws/aws-cdk/pull/25316 https://github.com/aws/aws-cdk/pull/25318/files, thanks for being vague in this discussion.

**Reply by kmcclive** [reply]
Thanks!
